.pre-commit-config.yaml

```diff
@@ -1,27 +1,23 @@
+default_stages: [commit, manual]
+
 exclude: ^kedro_mlflow/template/project/run.py
 repos:
     - repo: https://github.com/psf/black
       rev: 23.3.0
       hooks:
           - id: black
             language_version: python3.10
-    - repo: https://github.com/timothycrosley/isort
-      rev: 5.12.0
+    - repo: https://github.com/astral-sh/ruff-pre-commit
+      # Ruff version.
+      rev: v0.0.277
       hooks:
-          - id: isort
-    - repo: https://github.com/pycqa/flake8
-      rev: 6.0.0
-      hooks:
-          - id: flake8
-    - repo: https://github.com/asottile/pyupgrade
-      rev: v3.3.1
-      hooks:
-        - id: pyupgrade
-    - repo: https://github.com/asottile/blacken-docs
-      rev: v1.12.1
-      hooks:
-        - id: blacken-docs
-          additional_dependencies: [black==23.3.0]
```

Comment:
@noklam  I know the PR is closed but are you sure ruff replace ``blacken-docs``? This is a pre-commit to check black formatting for markdown file, so I think it it unrelated to other changes. I will restore it on the other PR if you agree?

Reply:
Oversight - feel free to restore this one.